rootfs/etc/profile.d/prompt.sh

```diff
@@ -28,12 +28,25 @@ function terraform_prompt() {
 # Define our own prompt
 PROMPT_HOOKS+=("geodesic_prompt")
 function geodesic_prompt() {
+       
+  case $PROMPT_STYLE in
+      plain)
+          # 8859-1 codepoints:
+          WHITE_HEAVY_CHECK_MARK=$(tput bold)$(tput setab 2)$'X'$(tput sgr0)' '
+          BLACK_RIGHTWARDS_ARROWHEAD=$'=> '
+          TWO_JOINED_SQUARES=$'¤ '  # perhaps § instead?
```

Comment:
Yea, I like `§`

Reply:
perhaps we should generalize the names? e.g. it's no longer "Two joined names"

Reply:
+1

I'll try my hand at some more generic sounding names?